_posts/2022-09-21-hvr-and-hva.markdown

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+font: serif
+title: HVR and HVA
+date: 2022-08-08
+highlight: monokai
+author: Matt Carter, Jack Menzie, Leon Ticharwa
+image: /img/blog/hvr-and-hva/banner.jpg
+tags: hvr hva data replication
+---
+
+# HVR and HVA
+
+## HVR
+
+![Architecture](/img/blog/hvr-and-hva/hvr-architecture.png)
+
+### HVR Hub Machine
+
+### Change Data Capture
+
+HVR and HVA utilise a log-based Change Data Capture (**CDC**) method to replicate data between both heterogeneous and homogeneous databases. A log-based capturing method involves maintaining a transaction log that records the content and metadata of a database, in which only incremental changes are replicated from source to target after the initial synchronisation of data. Through utilisation of a log-based CDC method, the following benefits are achieved:
+
+- Low latency
+- No requirements required for changes to made either programmatically or to the schema of databases
+- Minimal impact to the database itself
```

Comment:
I think its worth noting during the intial load, CDC it not used, rather traditional querying

Server impact can be minimised by segentation and pararalel processing (core dependant)